### DISCUSSION TITLE
Every file is opened in the same tab. How to open files in new tabs instead?

### DISCUSSION THREAD

**Discussion by faxotherapy** [pulsar-edit/.github#147] [Pulsar Support] [answered]
Hi, files are opened in the same tab. I can't find the option to open every file into new tab instead of same tab. I only see “Open in New Window” option. Also, would it be possible to configure the default option to open in new tabs instead. Thanks for your help.

**Comment by confused-Techie** [comment, marked as answer]
So most likely what's happening here is, when you open and view a file in Pulsar after single clicking it, when you click a new file it opens in the same tab. 

If you were to double click the file, or double click the file's tab itself, or even make changes within the file prior to clicking another then once you click on an additional file it'll open up in a new tab.

If you'd like to change this behavior in "Settings" > "Core" > "Allow Pending Pane Items" should be switched to off. 

Although a warning if you do, somewhat unintuitively this will mean you need to double click every file to open it, but would stop this behavior.
